### PR TITLE
Add support for ping4

### PR DESCRIPTION
**Pull request by sur5r**

### CODE REVIEW

**Review by akinomyoga — APPROVED** [review]

File: completions/ping [hunk 1]
```diff
@@ -65,8 +65,9 @@ _ping()
     fi
 
     [[ $1 == *6 ]] && ipvx=-6
+    [[ $1 == *4 ]] && ipvx=-4
     _known_hosts_real ${ipvx-} -- "$cur"
 } &&
-    complete -F _ping ping ping6
+    complete -F _ping ping ping4 ping6
```
Comment (scop) [inline completions/ping:71]: We should add a `ping4` symlink so the completion gets actually loaded for it, similarly as for `ping6`. For reference:
```
$ git grep -wFn ping6 | head -n 3
completions/.gitignore:154:/ping6
completions/Makefile.am:686:	ping6 \
completions/Makefile.am:961:		ping6
```

**Review by scop — CHANGES_REQUESTED** [review]

**Comment by sur5r** [comment]
Ah, truel, I missed that. Will amend the PR later today.

**Comment by akinomyoga** [comment]
Also, actually, the commit message should be in Conventional Commits (as described in CONTRIBITING.md).

File: completions/ping (hunk 1 above)
Comment (sur5r) [reply completions/ping:71]: Done.

**Review by scop — APPROVED** [review]

**Comment by scop** [comment]
Thanks!

**Pull request merged**